ci/build_windows.py

```diff
@@ -218,6 +246,8 @@ def main():
             os.environ["OpenCV_DIR"] = "C:\\Program Files\\OpenCV-v3.4.1\\build"
         if 'CUDA_PATH' not in os.environ:
             os.environ["CUDA_PATH"] = "C:\\Program Files\\NVIDIA GPU Computing Toolkit\\CUDA\\v9.2"
+        if 'MKL_ROOT' not in os.environ:
+            os.environ["MKL_ROOT"] = "C:\Program Files (x86)\IntelSWTools\compilers_and_libraries\windows\mkl"
```

Comment:
Not sure about the effects of the escape characters here. But for the path string,  I think using a raw string prefix flag (r'the/path/string'), in this case, is better.

Reply:
we test both format should be ok.  anyway change it now. thanks